pkg/metadata/inventories/inventories_test.go

```diff
@@ -159,6 +157,7 @@ func TestGetPayload(t *testing.T) {
 	}}
 
 	SetAgentMetadata("test", true)
+
 	SetCheckMetadata("check1_instance1", "check_provided_key1", 123)
 	SetCheckMetadata("check1_instance1", "check_provided_key2", "Hi")
 	SetCheckMetadata("non_running_checkid", "check_provided_key1", "this_should_be_kept")
```

Comment:
Would it make sense to update `"this_should_be_kept"`?

Reply:
indeed